src/main/flight/position_control.h

```diff
@@ -25,3 +25,6 @@ void positionControlInit(const positionControlConfig_t *config);
 bool isBelowLandingAltitude(void);
 
 const pidCoefficient_t *getAltitudePidCoeffs(void);
+void resetAltitudeControl (void);
+void altitudeControl(float targetAltitudeCm, float taskIntervalS, float verticalVelocity, float targetAltitudeStep);
```

Comment:
`altitudeControl()` and `resetAltitudeControl()` into line 24

Reply:
done